src/Microsoft.ComponentDetection.Orchestrator/Services/DetectorRestrictionService.cs

```diff
@@ -20,26 +20,33 @@ public IEnumerable<IComponentDetector> ApplyRestrictions(DetectorRestrictions re
     {
```

Comment:
Do not make any changes to this file, at least not in this PR. We are able to enable disabled detectors with the arguments, default scan doesn't have any `DetectorsFilter`. We can do a follow up PR just for this.